packages/core/nuxt-theme-module/theme/layouts/default.vue

```diff
@@ -3,16 +3,13 @@
     <LazyHydrate when-visible>
       <TopBar class="desktop-only" />
     </LazyHydrate>
-    <LazyHydrate when-idle>
-      <AppHeader />
-    </LazyHydrate>
+
+    <AppHeader />
 
     <div id="layout">
       <nuxt :key="$route.fullPath"/>
 
-      <LazyHydrate when-visible>
-        <BottomNavigation />
-      </LazyHydrate>
+      <BottomNavigation />
```

Comment:
There were some issues with LazyHydrate?

Reply:
Yes, but issue will likely be resolved in https://github.com/vuestorefront/vue-storefront/issues/6365